build/tutorials/platform/getting-started-for-users.md

```diff
@@ -0,0 +1,121 @@
+
+## Explorers
+
+Explorers are websites that index and present network activity, where you can look up individual transactions, and find out more about what's flowing through the network.
+
+### Official explorer
+
+[explorer.avax.network](https://explorer.avax.network/) is the official network explorer maintained by Ava Labs.
+
+### AvaScan
+
+[AvaScan](https://avascan.info/) is an independent explorer website, known for its slick presentation and comprehensive overview, especially interesting for [validators and delegators](https://avascan.info/staking/validators) as it shows lots of interesting information about individual network validators.
+
+### VScout
+
+[VScout](https://vscout.io/) is another alternative explorer for Avalanche. Among other things there you can see distribution of validators across the planet.
+
+## Dapps
+
+### Avalanche Bridge
+
+Once you have your browser extension set up, you are ready to interact with dapps on Avalanche. Most of what you will want to do (*yield farming*), will require that you have tokens other than AVAX. If you have those tokens on Ethereum (or an exchange that can send them to Ethereum), one of the cheapest and fastest ways of bringing them over is the [Avalanche Bridge](https://bridge.avax.network/).
+
+You can find a collection of video tutorials on the basic usage of the Avalanche Bridge [here](https://www.youtube.com/playlist?list=PLRHl-ulWK4-FPRA7SS1OrCOC8cOc2K8sP). Also, make sure that you go over the [FAQ](https://docs.avax.network/learn/avalanche-bridge-faq) which answers most common questions about the bridge and highlights things to watch out for.
+
+When you bridge over $75 or more of assets, you will also be airdropped some AVAX to help pay for initial swaps. Use those funds only to acquire additional AVAX as you will need AVAX to pay for fees on every other dapp you use! If you get stranded without enough AVAX for fees you will be unable to do anything else, so make sure you always have some AVAX in your wallet. You can swap for AVAX on:
+
+### Pangolin
+
+Another popular way of acquiring tokens on Avalanche are decentralized exchanges (DEXes). Biggest by liquidity is [Pangolin](https://app.pangolin.exchange/#/swap). It was launched with the help from Ava Labs, but it is completely community owned [DAO](https://gov.pangolin.exchange/) now.
+
+There is a collection of [tutorials](https://pangolin.exchange/tutorials/) explaining usage, and there is a community [Discord](https://discord.com/invite/PARrDYYbfw) too where you can find support and discussions.
+
+### BenQI
+
+Another popular DeFi dapp that recently launched is [BenQi](https://www.benqi.fi/), a decentralized liquidity protocol (basically, an application where you can lend and borrow tokens). You can find the instructions for using it [here](https://docs.benqi.fi/). 
```

Comment:
if you're going to mention benqi,

I would state it's benefits; 
you are able to supply your AVAX for a over-collateralized loan of up to 40% of your deposit.
Those funds enter your wallet for you to use unrestricted just like any other token.

Reply:
removed specific projects.